HW 8

orionst opened the pull request:
* По дизайну: не смог "подружить" BottomNavView с fab из фрагмента и со списком фото (панель 
   закрывает низ самого нижнего ряда.)
* Замечания не все исправил, буду исправлять по мере того как найду
  верное решение :/
* Активити, отвечающую за просмотр фото, подчистил. Вызывается по тапу
  на изображение.

orionst committed:
* По дизайну: не смог "подружить" BottomNavView с fab из фрагмента и
  со списоком фото (панель закрывает низ самого нижнего ряда.)
* Замечания не все исправил, буду исправлять по мере того как найду
  верное решение :/
* Активити, отвечающую за просмотр фото, подчистил. Вызывается по тапу
  на изображение.

grigoryfedorov commented on app/src/main/java/com/orionst/mymaterialdesignapp/MainActivity.java:
```diff
@@ -50,6 +54,26 @@ protected void onCreate(Bundle savedInstanceState) {
 
         TabLayout tabLayout = (TabLayout) findViewById(R.id.tabs);
         tabLayout.setupWithViewPager(mViewPager);
+        tabLayout.addOnTabSelectedListener(new TabLayout.OnTabSelectedListener() {
```
это можно убрать, если есть setupWithViewPager

orionst committed:
* Исправление замечания
  https://github.com/orionst/Photogramm/pull/7/files/8d531804d9b911f8d84d754bff8d25686839eb49#r216170191

orionst committed:
* Исправление замечаний:
  - Нижний тулбар теперь не скрывает низ последнего ряда фото.
  - DB photos fragment теперь принудительно добавляется только при первом
  вызове контейнера фрагментов.

orionst committed:
Приведение дизайна проекта в полное соответвие требованиям ДЗ.
